src/main/xar-resources/data/upgrading.xml

```diff
@@ -17,207 +17,85 @@
+            <para>For the same reason, you should not install the new version of eXist over the previous version. Instead, these instructions have you install the new version into a fresh new directory.</para>
+        </section>
+        <section id="binary-compatible-upgrades">
+            <title>Binary Compatible Upgrades</title>
+            <para>Follow these steps if the new version of eXist is binary-compatible with the your old version. Check the new version's release notes to find out whether it is binary compatibility with your old version.</para>
             <procedure>
                 <step>
-                    <para>Create a <ulink url="backup.xml">backup</ulink> of your data. If the new
-                        version is <emphasis>binary compatible</emphasis> with the old version, keep
-                        the data directory (by default in <filename>webapp/WEB-INF/data</filename>)
-                        of the old version. Note: A running instance of eXist-db needs to be stopped before copying files
-                        from the data dir.</para>
+                    <para>Stop the old installation of eXist, if it is running.</para>
+                </step>
+                <step>
+                    <para>Create a <ulink url="backup.xml">backup</ulink> of the data from your old installation of eXist.</para>
+                </step>
+                <step>
+                    <para>Install the new version of eXist. Do not install one eXist version over another one. Always install the new version into a fresh new directory.</para>
                 </step>
                 <step>
-                    <para>Install the new version into a different location.</para>
+                    <para>Remove the <emphasis>empty</emphasis> <code>data</code> directory from the new installation. Typically the data directory is located in <code>webapp/WEB-INF/data</code>.<!-- TODO Add fool-proof procedures for locating your data directory for different installations. For example, macOS  users who install eXist via disk image installer need to look in ~/Library/Application Support/org.exist/ - JCW --></para>
                 </step>
                 <step>
-                    <para>If the new version is <emphasis>binary compatible</emphasis>, replace the
-                        data directory of the new install with the one from the old one.</para>
-                    <para>Otherwise you need to do a full <ulink url="backup.xml#restore">restore</ulink> of the data.</para>
+                    <para>Copy the entire data directory of the old version into the corresponding location of the new version.</para>
```

Comment:
This is ambigous, corresponding location implies that ``old/data`` goes to ``new/data`` actually ``old/data`` must go to ``new/just-deleted-in-previous-step``